src/stores/geometryLogStore.ts

```diff
@@ -0,0 +1,41 @@
+import { action, computed, observable } from 'mobx';
+import ILogEntry from '~/models/ILogEntry';
+import logActions from '~/enums/logActions';
+import entityNames from '~/enums/entityNames';
+
+export class GeometryLogStore {
+    @observable private _log: ILogEntry[];
+
+    constructor() {
+        this._log = [];
+    }
+
+    @computed get log(): ILogEntry[] {
```

Comment:
get logs

Reply:
Or would "events" be better?

geometryLogStore -> geometryEventStore?

Reply:
We have one log, multiple log entries. We don't have multiple logs, that would mean we have multiple different logs to store different things. Log vs Event is debatable, event could in this context indeed work better.

Reply:
events